test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -536,6 +536,14 @@ class test_ucp_tag_nbx : public test_ucp_tag {
 
     bool enable_proto() const
     {
+        const char *proto = getenv("UCX_PROTO_ENABLE");
+        int dest;
+
+        if ((proto != NULL)) {
+            /* Handle external proto v2 enabling */
+            return ucs_config_sscanf_bool(proto, &dest, NULL);
+        }
+
```

Comment:
need to check m_ucp_config
see https://github.com/openucx/ucx/pull/8770/files#diff-e792a43201767a6ff9e81af8de60b40e6df038ff7e85325187b1097115eb222cR40